java/test/org/openqa/selenium/remote/TracedCommandExecutorTest.java

```diff
@@ -76,6 +76,22 @@ public void canCreateSpanWithAllAttributes() throws IOException {
     verifyNoMoreInteractions(span);
   }
 
+  @Test
+  public void canCreateSpanFromNullParameter() throws IOException {
+    SessionId sessionId = new SessionId(UUID.randomUUID());
+    Map<String, Object> parameters = new HashMap<>();
+    parameters.put("param1", null);
+    Command command = new Command(sessionId, "findElement", parameters);
+
+    tracedCommandExecutor.execute(command);
+
+    verify(span, times(1)).setAttribute("sessionId", sessionId.toString());
+    verify(span, times(1)).setAttribute("command", "findElement");
+    verify(span, times(1)).setAttribute("parameter.param1", String.valueOf((Object) null));
```

Comment:
Subtlety between the test and the code above is the cast to `Object` that is done.

forgive me for using https://www.tutorialspoint.com/compile_java_online.php
but is was the quickest way for me to _test_ this as I am not on my laptop.

```
public class HelloWorld{
     public static void main(String []args){
        System.out.println("Hello World");
        System.out.println(String.valueOf((Object) null));
        System.out.println((String) null);
        System.out.println(String.valueOf((null)));
     }
}
```

yields:
```
Hello World
null
null
Exception in thread "main" java.lang.NullPointerException
	at java.lang.String.<init>(String.java:166)
	at java.lang.String.valueOf(String.java:3008)
	at HelloWorld.main(HelloWorld.java:7)
```

Reply:
Thx. Removed unless cast and code.